setup.json

```diff
@@ -87,12 +86,12 @@
     },
     "include_package_data": true,
     "install_requires": [
-	"aiida-core[atomic_tools] >= 1.2.1,!=1.4.0,!=1.4.1,<2",
-	"psycopg2-binary~=2.8.3",
-	"subprocess32",
-	"pymatgen>=2019.7.2,<=2022.02.03,!=2019.9.7",
-	"lxml",
-	"packaging",
-	"parsevasp@git+https://github.com/aiida-vasp/parsevasp@058bb26872588c4a83dbc93afcfbf638963be1fc"
+		"aiida-core[atomic_tools] >= 1.2.1,!=1.4.0,!=1.4.1,<2",
+		"psycopg2-binary~=2.8.3",
+		"subprocess32",
```

Comment:
Do not think we need that now as we now are strictly Python 3.

Reply:
I think so, we should drop python 2 support (I think we already do?).

Reply:
Done @espenfl

Reply:
Hmmm it turns out that if you remove that package, the pre-commit goes crazy. Any idea why?

Reply:
One of the precommit hooks relied on it. I think it is the one for synching version numbers. Maybe we should update it?

Reply:
Done, please update @JPchico and remove package. Should be fine now. Thanks.